js-miniapp-bridge/src/modules/utility-manager.ts

```diff
@@ -46,4 +46,19 @@ export class UtilityManager {
       );
     });
   }
+
+  /**
+   * Trigger launchAppSettings from host
+   * @returns true or false whether launch app settings is launched or not
+   */
+  launchAppSettings() {
+    return new Promise<boolean>((resolve, reject) => {
+      return this.executor.exec(
+        'launchAppSettings',
+        null,
+        response => resolve(MiniAppBridgeUtils.BooleanValue(response)),
+        error => reject(parseMiniAppError(error))
+      );
+    });
```

Comment:
Thanks for the change.

We don't need to add the following in common-bridge.ts ?

  launchAppSettings() {
    return this.utilityManager. launchAppSettings();
  }